### PR TITLE
Do not overwrite _measurement tag for multi-tenant reads

### PR DESCRIPTION
**Pull request by stuartcarnie**

### CODE REVIEW

File: services/storage/series_cursor.go [hunk 1]
```diff
@@ -181,12 +183,16 @@ RETRY:
...
-		c.filterset = mapValuer{"_name": string(sr.Name)}
+		c.filterset = make(mapValuer)
 		for _, tag := range c.tags {
 			c.filterset[string(tag.Key)] = string(tag.Value)
 		}
 
-		c.tags.Set(measurementKey, sr.Name)
+		if !c.multiTenant {
+			c.filterset["_name"] = string(sr.Name)
+			c.tags.Set(measurementKey, sr.Name)
```
Comment (nathanielc) [inline services/storage/series_cursor.go]: If it is multiTenant is the mesurementKey already part of the tags?

Comment (stuartcarnie) [reply services/storage/series_cursor.go]: exactly – which means less allocations and more efficient 🎉

Comment (stuartcarnie) [reply services/storage/series_cursor.go]: …and now we just need to get `_field` in there

Comment (nathanielc) [reply services/storage/series_cursor.go]: Makes sense

**Pull request merged**